src/ports/rt-kernel/sampleapp_main.c

```diff
@@ -194,6 +194,7 @@ int main (void)
    pnet_default_cfg.cb_arg = (void *)&appdata;
    strcpy (pnet_default_cfg.station_name, appdata.arguments.station_name);
    strcpy (pnet_default_cfg.file_directory, APP_DEFAULT_FILE_DIRECTORY);
+   /* Note: threads_cfg not is used for rt-kernel (initialized to 0) */
```

Comment:
This should be fixed.

Reply:
Assert in rt-kernel code to make sure it not is set.

Reply:
Implemented operating-system dependent pnal configuration.